client/onboarding/add-business-info-task/index.js

```diff
@@ -0,0 +1,171 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useState, useContext } from 'react';
+import { __ } from '@wordpress/i18n';
+import { SelectControl, Card, CardBody } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import WizardTaskItem from 'wcpay/additional-methods-setup/wizard/task-item';
+import WizardTaskContext from 'wcpay/additional-methods-setup/wizard/task/context';
+import OnboardingSelectControl from 'wcpay/components/onboarding-select-control';
+import { useBusinessTypes } from 'wcpay/data/onboarding';
+import strings from '../strings';
+
+const AddBusinessInfo = () => {
+	const { setCompleted } = useContext( WizardTaskContext );
+	const { businessTypes, isLoading } = useBusinessTypes();
+	const {
+		connect: { availableCountries, country },
+	} = wcpaySettings;
+
+	const countries = Object.keys( availableCountries ).map(
+		( countryCode ) => {
+			return {
+				label: availableCountries[ countryCode ],
+				value: countryCode,
+			};
+		}
+	);
+
+	const [ businessCountry, setBusinessCountry ] = useState( country );
+	const [ businessType, setBusinessType ] = useState();
```

Comment:
To avoid `OnboardingSelectControl` to throw a warning, you'll need to provide a default value here. Otherwise it will complain about changing a uncontrolled prop to controlled. That's because using an empty `useState` here results in it being `undefined`.

Reply:
As we discussed on Slack, we needed to use `''` as the placeholders, to prevent the warnings getting thrown.